packages/medusa/src/api/routes/admin/orders/create-reservation-for-line-item.ts

```diff
@@ -20,22 +20,10 @@ import {
  *       schema:
  *         $ref: "#/components/schemas/AdminOrdersOrderLineItemReservationReq"
  * x-codeSamples:
- *   - lang: JavaScript
- *     label: JS Client
- *     source: |
- *       import Medusa from "@medusajs/medusa-js"
- *       const medusa = new Medusa({ baseUrl: MEDUSA_BACKEND_URL, maxRetries: 3 })
- *       // must be previously logged in or use api token
- *       medusa.admin.orders.createReservation(order_id, line_item_id, {
- *         location_id
- *       })
- *       .then(({ reservation }) => {
- *         console.log(reservation.id);
- *       });
```

Comment:
Removed this snippet as it's not actually added to the JS Client and according to Phil the endpoint might be removed in the next release